Code/GraphMol/MolDraw2D/test1.cpp

```diff
@@ -4533,6 +4533,12 @@ void test23JSONAtomColourPalette() {
     TEST_ASSERT(text.find("fill:#3366E5") != std::string::npos);
     TEST_ASSERT(text.find("fill:#E59900") != std::string::npos);
 #endif
+    std::regex regex(
+        R"regex(path d='M \d+.\d+,\d+.\d+ L \d+.\d+,\d+.\d+ L \d+.\d+,\d+.\d+.*)regex");
```

Comment:
I don't think you need the `*` at the end of this, but I'm certainly no regexpert

Reply:
I guess it might even be unhelpful.